### ISSUE TITLE
Too many request error

### ISSUE THREAD

**Issue by tanishqdiwan111** [trailofbits/Codex-Decompiler#5]
i have installed plugin in ghidra after pressing ctrl+j clicking on the function , it shows the too many request error

**Comment by ret2libc** [comment]
Hi @tanishqdiwan111 ! I think that error is due to the number of calls you are making to OpenAI. Does the error always happen? Did you try after a bit to see if it disappeared?

**Comment by tanishqdiwan111** [comment]
yes this error always happen and i have tried after few minutes but it always gives too many request error 
sometimes ghidra pops up with "multiple actions have been mapped to ctrl+j actions that can be enabled at the same time should be mapped to different keys" window

**Comment by parikhakshat** [comment]
Hi @tanishqdiwan111, AFAIK the "multiple actions have been mapped to ctrl+j actions" error happens when you run the script from the script manager multiple times. You can restart your ghidra instance and run it just once. Also, the "too many request" error happens when your openai api_key has reached its usage limit. Try creating a new key or a key from a new account.

**Comment by tanishqdiwan111** [comment]
i tried a new key with new openai account but it still showing error 
Collecting data about function.
Sending data to OpenAI api.
Error in accessing api: Too Many Requests
Invalid output from api
i also updated this in bashrc "export OPENAI_API_KEY="with my key"
